### PR TITLE
Added information about new CarPlay Apps

### PR DESCRIPTION
**Pull request by below**

### CODE REVIEW

**Comment by below** [comment]
Added more information about CarPlay

File: README.md [hunk 1]
```diff
@@ -470,10 +471,39 @@ _Daniel Miao_
...
-
-
 
+## Accelerate your app with CarPlay
+
+https://developer.apple.com/wwdc20/10635 
+
+_Jonathan Hersh, Allen Langmaier_
+
+- **New CarPlay Templates***
```
Comment (rogerluan) [inline README.md]: `**New CarPlay Templates***`VS `**New Templates**` is confusing to me, they sound like the same thing

File: README.md [hunk 2]
```diff
@@ -470,10 +471,39 @@ _Daniel Miao_
...
+- **New Templates**
+  - Communication Apps
+    - `CPMessageListItem`
+    - Contact Template
+  - List Template
+  - Tab Bar Template
+  - Now playing template
+  - Point of Interest Template
+  - Information Template
+- **CarPlay Design Priciples**
```
Comment (rogerluan) [inline README.md]:
```suggestion
- **CarPlay Design Principles**
```

**Review by rogerluan — CHANGES_REQUESTED** [review]
Aside from the comments below, there're capitalization inconsistencies in several places (e.g. `template` VS `Template`), maybe give that a second glance 👀 

I <3 CarPlay 🚀 looking forward to experiencing the future!

File: README.md (hunk 1 above)
Comment (below) [reply README.md]: Should be fixed

File: README.md [hunk 3]
```diff
@@ -77,7 +78,8 @@ https://developer.apple.com/wwdc20/101/
   - Brand new **cycling** option to reduce your carbon footprint
   - **EV-Routing** option to optimize your route for electric cars - e.g. by selecting routes with charging station on your way
 - **CarPlay**
-  - Add custom wallpapers
+  - New wallpaper option
+  - New Categories for CarPlay Apps: Parking, EV-Charging, and Quick-Food ordering
```
Comment (rogerluan) [inline README.md]:
```suggestion
  - New categories for CarPlay apps: Parking, EV-Charging, and Quick-Food ordering
```

Last one, promise 🙈

**Review by rogerluan — CHANGES_REQUESTED** [review]

**Review by Blackjacx — APPROVED** [review]
Thanks for your addition 👍

**Pull request merged**